aptos-move/framework/aptos-framework/sources/configs/consensus_config.move

```diff
@@ -24,14 +23,13 @@ module aptos_framework::consensus_config {
     }
 
     /// This can be called by on-chain governance to update on-chain consensus configs.
-    public fun set(account: &signer, config: vector<u8>) acquires ConsensusConfig {
+    public fun set(account: &signer, config: vector<u8>) {
         system_addresses::assert_aptos_framework(account);
         assert!(vector::length(&config) > 0, error::invalid_argument(EINVALID_CONFIG));
+        std::config_for_next_epoch::upsert<ConsensusConfig>(account, ConsensusConfig {config});
+    }
 
-        let config_ref = &mut borrow_global_mut<ConsensusConfig>(@aptos_framework).config;
-        *config_ref = config;
-
-        // Need to trigger reconfiguration so validator nodes can sync on the updated configs.
-        reconfiguration::reconfigure();
```

Comment:
@zekun000 @movekevin it's now the other way around: `terminate_slow_reconfigure()` calls `consensus_config::on_new_epoch()`. The module dependency also changed. Any compatibility issue?

Reply:
the easiest way is probably to run a framework upgrade test to see if it can pass